.travis.yml

```diff
@@ -88,7 +88,7 @@ script:
           -DKokkos_ENABLE_TESTS=On
           ${CMAKE_BUILD_TYPE:+-DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}} &&
     make VERBOSE=1 -j2 &&
-    travis_wait make test CTEST_OUTPUT_ON_FAILURE=1 &&
+    travis_wait 60 make test CTEST_OUTPUT_ON_FAILURE=1 &&
```

Comment:
I am not very comfortable with merging that in.  @junghans Would you remind me why this is there in the first place?
@masterleinad What command is taking so long?  Is it while compiling?  linking?  running the tests?


For reference [here](https://docs.travis-ci.com/user/common-build-problems/#build-times-out-because-no-output-was-received) is the doc.

Reply:
`PerformanceTest_OpenMP` and ` PerfTestExec`. Running with verbose output should likely also do the trick.

Reply:
Hmm, 60 min wall time seems a bit long....

Reply:
Blocked by this change.

> `PerformanceTest_OpenMP` and ` PerfTestExec`. Running with verbose output should likely also do the trick.

Please give it a try.